packages/ui-components/src/components/Matrix/MatrixHeader.tsx

```diff
@@ -23,6 +24,18 @@ const THead = styled(TableHead)`
   top: 0;
   z-index: 3;
 `;
+
+const CellLink = styled(RouterLink)`
+  color: white;
+  text-decoration: none;
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+const TitleCell = ({ title, entityLink }: { title: string; entityLink?: string }) => {
+  return entityLink ? <CellLink to={entityLink}>{title}</CellLink> : <>{title}</>;
```

Comment:
This is very nitpicky but personally I like to do this instead of a ternary to improve readability:
```
if(entityLink)
  return <CellLink to={entityLink}>{title}</CellLink>;
return <>{title}</>
```

I'm also wondering if the fragment around `title` is needed?